app/controllers/api/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Api::CommentsController < ApplicationController
+  def index
+    @comments = Comment.all
+    render json: @comments
+  end
+
```

Comment:
![image](https://github.com/Benawi/Blog-App/assets/93039359/cc4a403c-2544-4f74-8660-923e19b72e58)

- [ ] Great work on implementing the comments API 👍 but again same thing here you are supposed to get the comments for that specific post_id which you have in your URL and it is also mentioned in the project requirements 😊